styles/helpers/inputs.css

```diff
@@ -140,4 +140,56 @@ title: Form
       @mixin input var(--color-input-background), var(--color-body), var(--color-input-disabled), var(--color-input-error), var(--color-input-error), var(--color-input-error);
     }
   }
+
+  /*---
+  section: Forms
+  title: Input group
+
+  ```example:html
+  <form class="form">
+    <div class="input-group">
+      <input type="text" placeholder="Enter your email">
+      <button class="input-group-btn button">Sign Up</button>
```

Comment:
spans for addons like currency or things you'd wnat to suffix with.  See [Bootstrap](http://v4-alpha.getbootstrap.com/components/input-group/#basic-example)